docs: Add instructions to configure TEE Paratime Client Node (#915)
oasisprotocol/docs: docs/node/run-your-node/paratime-client-node.mdx

:::

> ## Configuring TEE Paratime Client Node

Reviewer: I don't see an explanation why or when one would want to do this. We should add it and maybe reword when this would be required -- probably something along the lines that this is needed if one wants a node that can issue queries that access confidential data.

If your node requires the ability to issue queries that can access confidential
data, start by following the [Configuration](#configuration) section to create
the `/node/etc/config.yml` file. Once the file is set up, add the following
content to the `runtime` part within the `/node/etc/config.yml` file:

> 
> **Update Runtime Configuration:**

Reviewer: ```suggestion
```

   ```yaml
   runtime:
       # Paths to ParaTime bundles for all of the supported ParaTimes.
       paths:
           - {{ runtime_orc_path }}
       environment: sgx
       config:
         {{ paratime_id }}:
             estimate_gas_by_simulating_contracts: true
             allowed_queries:

>                  - all_expensive: true 

Reviewer: This is unrelated to running "TEE client nodes", rather it is required in case you want to issue certain runtime queries (e.g. for running the Web3 Gateway). I'm not sure if this should be here, as we already mention this in the [Web3 Gateway chapter](https://docs.oasis.io/node/web3#oasis-paratime-client-node)? We could leave it in this chapter, but move it outside the "TEE client nodes" section, rather just have a separate section about "Enabling expensive queries".

       sgx_loader: /node/bin/oasis-core-runtime-loader

>     ```

Reviewer: Drop the extra indentation of this block.

Before using this configuration you should collect the following information to

> replace  the  variables present in the configuration file:

Reviewer: ```suggestion
replace the variables present in the configuration file:
```

* `{{ paratime_id}}`: You can find the `paratime_id` in the Network Parameters

> page ([Mainnet], [Testnet])

Reviewer: ```suggestion
chapter ([Mainnet], [Testnet])
```

:::

## Configuring TEE Paratime Client Node

If your node requires the ability to issue queries that can access confidential
data, start by following the [Configuration](#configuration) section to create
the `/node/etc/config.yml` file. Once the file is set up, add the following
content to the `runtime` part within the `/node/etc/config.yml` file:

```yaml
# ... sections not relevant are omitted ...
runtime:
    # Paths to ParaTime bundles for all of the supported ParaTimes.
    paths:
        - {{ runtime_orc_path }}
    environment: sgx
    sgx_loader: /node/bin/oasis-core-runtime-loader
```

Before using this configuration you should collect the following information to
replace the variables present in the configuration file:

* `{{ paratime_id}}`: You can find the `paratime_id` in the Network Parameters

> chapter ([Mainnet], [Testnet])

Reviewer: This does not seem to be used anywhere.

* `{{ runtime_orc_path }}`: Path to the
[ParaTime bundle](paratime-client-node.mdx#the-paratime-bundle) of the form
`/node/runtimes/foo-paratime.orc`.

## Enabling Expensive Queries

In case you need to issue runtime queries that may require more resources to
compute (e.g. when running a Web3 Gateway), you need to configure the following
in your node's `/node/etc/config.yml` file:

```yaml
# ... sections not relevant are omitted ...
runtime:
    paths:
      - {{ runtime_orc_path }}
    environment: sgx
    config:

>       {{ paratime_id }}:

Reviewer: Above you use `runtime` here you use `paratime`. It should be consistent (also with the above section).

:::

## Configuring TEE Paratime Client Node

If your node requires the ability to issue queries that can access confidential
data, start by following the [Configuration](#configuration) section to create
the `/node/etc/config.yml` file. Once the file is set up, add the following
content to the `runtime` part within the `/node/etc/config.yml` file:

```yaml
# ... sections not relevant are omitted ...
runtime:
    # Paths to ParaTime bundles for all of the supported ParaTimes.
    paths:
        - {{ runtime_orc_path }}
    environment: sgx
    config:
      {{ paratime_id }}:

>           estimate_gas_by_simulating_contracts: true

Reviewer: This also needs to be moved below, it goes hand-in-hand with expensive queries and is unrelated to TEE client nodes.

    sgx_loader: /node/bin/oasis-core-runtime-loader
```

Before using this configuration you should collect the following information to
replace the variables present in the configuration file:

* `{{ paratime_id}}`: You can find the `paratime_id` in the Network Parameters
chapter ([Mainnet], [Testnet])
* `{{ runtime_orc_path }}`: Path to the
[ParaTime bundle](paratime-client-node.mdx#the-paratime-bundle) of the form
`/node/runtimes/foo-paratime.orc`.

> ## Enabling expensive queries

Reviewer: ```suggestion
## Enabling Expensive Queries

```

> To enable expensive querries set `all_expensive` parameter to `true` in the
> `runtime` part within the `/node/etc/config.yml` file:

Reviewer: ```suggestion
In case you need to issue runtime queries that may require more resources to
compute (e.g. when running a Web3 Gateway), you need to configure the following
in your node's `/node/etc/config.yml` file:
```
